### PR TITLE
Modernise several formula

### PR DESCRIPTION
**Pull request by afh**

### CODE REVIEW

**Comment by tdsmith** [comment]
`diction` test and audit are failing; the test failure is probably because the decimal separator is `.` in the test bot's locale. That particular test should be more concise; I'm probably happy if `echo This is a test of GNU diction. | style` (or its moral equivalent) produces output including `7 words`.

**Comment by afh** [comment]
@tdsmith Thanks for you feedback, I've added `ENV["LC_ALL"] = "C"` to the test and updated the expected output accordingly.

I find having a more extensive test as an example helpful in roughly judging what the software does.

File: Library/Formula/diction.rb [hunk 1]
```diff
@@ -1,14 +1,68 @@
...
+    # http://www.bartleby.com/141/strunk1.html
+    file = "strunk_intro.txt"
+    (testpath/file).write <<-EOS.undent
+    It is an old observation that the best writers sometimes disregard the
+    rules of rhetoric. When they do so, however, the reader will usually find in
+    the sentence some compensating merit, attained at the cost of the
+    violation. Unless he is certain of doing as well, he will probably do best
+    to follow the rules. After he has learned, by their guidance, to write
+    plain English adequate for everyday uses, let him look, for the secrets of
+    style, to the study of the masters of literature.
```
Comment (mikemcquaid) [inline Library/Formula/diction.rb]: Ident this one further.

File: Library/Formula/diction.rb [hunk 2]
```diff
@@ -1,14 +1,68 @@
...
+            subordinating conjunction (2) conjunction (0) preposition (0)
+    EOS
+    assert_equal style_result, shell_output("#{bin}/style #{file}")
+
+    diction_result =<<-EOS.undent
+#{file}:2: When [they] do [so], [however], the reader will usually find in the sentence some compensating merit, attained at the cost of the violation.
+
+#{file}:5: After he has learned, by their guidance, to write plain English [adequate] for everyday uses, let him look, for the secrets of style, to the study of the masters of literature.
+
+4 phrases in 4 sentences found.
```
Comment (mikemcquaid) [inline Library/Formula/diction.rb]: Ident this fully.

File: Library/Formula/diction.rb [hunk 3]
```diff
@@ -1,14 +1,68 @@
...
+    EOS
+    assert_equal diction_result, shell_output("#{bin}/diction #{file}")
```
Comment (mikemcquaid) [inline Library/Formula/diction.rb]: I don't think we need to match the contents exactly for two comments; it'll be a bit error prone. I think checking exit codes and/or partial output will be sufficient.

File: Library/Formula/ktoblzcheck.rb [hunk 4]
```diff
@@ -16,6 +14,16 @@ def install
...
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    result =<<-EOS.undent
+    Bank: 'Bundesbank' at 'Berlin' (10000000)
+    Account: 123456789
+    Result is: (0) Ok
+
```
Comment (mikemcquaid) [inline Library/Formula/ktoblzcheck.rb]: Ident this a line further.

File: Library/Formula/ktoblzcheck.rb [hunk 5]
```diff
@@ -16,6 +14,16 @@ def install
...
+    EOS
+    assert_equal result, shell_output("#{bin}/ktoblzcheck 10000000 123456789")
```
Comment (mikemcquaid) [inline Library/Formula/ktoblzcheck.rb]: Again, I think matching exact string output here will be potentially error-prone. Partial output and/or exit code checking seems enough.

File: Library/Formula/lout.rb [hunk 6]
```diff
@@ -1,20 +1,47 @@
...
+    @Doc @Text @Begin
+    @Display @Heading { Introduction by W. J. Harvey }
+    For Virginia Woolf, @I Middlemarch was 'the magnificent book which for all its
+    imperfections is one of the few English novels written for grown-up people.'
+    @PP
+    She was, no doubt, thinking of George Eliot's unblinking but compassionate
+    delineation of her characters, of the subtlety of psychological analysis and
+    the maturity of moral comment which underlie this complex and varied novel of
+    English provincial life in the early nineteenth century.
+    @End @Text
```
Comment (mikemcquaid) [inline Library/Formula/lout.rb]: Ident this further.

Comment (mikemcquaid) [inline Library/Formula/lout.rb]: And don't need this to be as long as it is, I think.

File: Library/Formula/lout.rb [hunk 7]
```diff
@@ -1,20 +1,47 @@
...
+
+               She was, no doubt, thinking of George Eliot's
+          unblinking but compassionate delineation of her characters,
+          of the subtlety of psychological analysis and the maturity
+          of moral comment which underlie this complex and varied
+          novel of English provincial life in the early nineteenth
+          century.
+\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n
+    EOS
+    assert_equal result, shell_output("#{bin}/lout -p #{input}")
```
Comment (mikemcquaid) [inline Library/Formula/lout.rb]: Same as above; partial output and exit codes.

File: Library/Formula/objc-run.rb [hunk 8]
```diff
@@ -1,14 +1,12 @@
...
+  homepage "https://github.com/iljaiwas/objc-run"
+  url "https://github.com/iljaiwas/objc-run/archive/1.4.tar.gz"
+  sha1 "86c15705359f2dc2c62d70e358caf63c2a468fee"
+  head "https://github.com/iljaiwas/objc-run.git"
 
   def install
-    bin.install 'objc-run'
-    (share+'objc-run').install 'examples', 'test.bash'
+    bin.install "objc-run"
+    (share+"objc-run").install "examples", "test.bash"
```
Comment (mikemcquaid) [inline Library/Formula/objc-run.rb]: Probably don't want to have any formulae with just string changes.

File: Library/Formula/qrencode.rb [hunk 9]
```diff
@@ -1,5 +1,3 @@
-require "formula"
-
```
Comment (mikemcquaid) [inline Library/Formula/qrencode.rb]: Similarly, probably don't want to change this as it'll show in `brew update` currently.

**Pull request closed without merging**